### PR TITLE
Fix optional type aliases

### LINKED ISSUE #257: Actix: type alias makes Option<T> required
Having a schema like this:
```rust
type Email = Option<String>;

#[derive(Apiv2Schema, Serialize)]
struct User {
    id: i32,
    username: String,
    email: Email,
}
```

The `email` field is optional, but in the generated spec it is shown as required (`"required":["email","id","username"]`)

Full code:
```rust
use actix_web::{App, HttpServer};
use paperclip::actix::web::{self, Json};
use paperclip::actix::{api_v2_operation, Apiv2Schema, OpenApiExt};
use serde::Serialize;

type Email = Option<String>;

#[derive(Apiv2Schema, Serialize)]
struct User {
    id: i32,
    username: String,
    email: Email,
}

#[api_v2_operation]
async fn example() -> Json<User> {
    Json(User {
        id: 1,
        username: "john".into(),
        email: None,
    })
}

#[actix_web::main]
async fn main() -> std::io::Result<()> {
    HttpServer::new(|| {
        App::new()
            .wrap_api()
            .route("/", web::get().to(example))
            .with_json_spec_at("/api/spec")
            .build()
    })
    .bind("127.0.0.1:8080")?
    .run()
    .await
}
```

Cargo.toml:
```toml
[dependencies]
actix-web = "3"
paperclip = { git = "https://github.com/wafflespeanut/paperclip", features = ["actix"] }
serde = { version = "1.0", features=["derive"] }
```

### PR DESCRIPTION
**Pull request by omid**
Fix #257

### CODE REVIEW

File: macros/src/actix.rs [hunk 1]
```diff
@@ -769,37 +769,23 @@ fn add_optional_impl(name: &Ident, generics: &Generics) -> proc_macro2::TokenStr
...
-        Type::Path(ref p) => {
-            let ty = p
-                .path
-                .segments
-                .last()
-                .expect("expected type for struct field");
-
-            if p.path.segments.len() == 1 && &ty.ident == "Option" {
-                is_required = false;
-            }
```
Comment (wafflespeanut) [inline macros/src/actix.rs:784]: This is certainly not what I expected in code. I guess you can't always remember how the code evolves.

File: core/src/v2/schema.rs [hunk 2]
```diff
@@ -312,6 +316,10 @@ impl<T> Apiv2Schema for Option<T> {
     default fn security_scheme() -> Option<SecurityScheme> {
         None
     }
+
+    fn is_required() -> bool {
```
Comment (wafflespeanut) [inline core/src/v2/schema.rs:320]: Can we make this a constant? It's a primitive value after all.

Comment (omid) [reply core/src/v2/schema.rs]: Done :)

**Comment by wafflespeanut** [comment]
One last thing. Can you add/modify a test for it?

**Review by wafflespeanut — APPROVED** [review]
Thanks!

**Pull request merged**